frontend/cypress/tests/main/blocks-calendar.cy.js

```diff
@@ -414,14 +432,10 @@ describe('Event Calendar Block Tests', () => {
     cy.get('#toolbar-save').click();
     cy.wait('@content');
     cy.url().should('eq', Cypress.config().baseUrl + '/my-page');
-    cy.get('.card-listing:first-of-type .image-wrapper .date-inset').should(
-      'not.have.class',
-      'has-end-date',
-    );
-    cy.get('.card-listing:nth-child(2) .image-wrapper .date-inset').should(
-      'have.class',
-      'has-end-date',
-    );
```

Comment:
Why did you remove the assertion about the multi-day event?

Reply:
instaed of that
`  cy.get('a[href="/my-second-event"]')
      .next('div.card-inner')
      .find('.date-inset')
      .should('not.have.class', 'has-end-date');
      `

Reply:
because we are only testing for daterange with same date and it doesn't make sense to check the other to don't have has-end-date. Maybe I should have not remove it. I will add it in next pr of cypress.